devel/poco/Portfile

```diff
@@ -5,7 +5,7 @@ PortSystem          1.0
 name                poco
 version             1.8.0.1
 categories          devel
-maintainers         nomaintainer
+maintainers         gmail.com:vandervelde.ag openmaintainer
```

Comment:
Please add your GitHub username to the maintainers line.

Reply:
hi! how do i add it, just @<username> on the same line?

Reply:
Yes. But you need to join it to your email like this:
`maintainers         {@arjanvandervelde gmail.com:vandervelde.ag} openmaintainer`

Reply:
thanks. done.